### DISCUSSION TITLE
Issue with Generating ML Reports for Observation in Release 5.0

### DISCUSSION THREAD

**Discussion by siddhishinde0723** [Sunbird-Lern/Community#188] [Query]
Hello Team,

In Release 5.0, I'm encountering an issue while attempting to generate ML reports for observation. Whenever I click on the reports, the screen continuously loads without displaying any content. I've attempted to debug the problem, but so far, I haven't been able to identify the cause of the issue.

I have attached a recording demonstrating the problem. 
`[

https://github.com/Sunbird-Lern/Community/assets/72926048/f7364ce8-3916-407d-91ce-bfe935575d38

]`

Could someone please provide guidance on how to resolve this issue?

cc: @aks30 @vijiurs @kiranharidas187 @varsha-sl @Prateek-slokam @aishwaryashikshalokam

**Comment by AmiableAnil** [comment]
@siddhishinde0723 may know which API is called here. Can you please share the logs here. By just looking the vide, we are not getting the context here.

cc: @PoojaJManjooran

**Reply by siddhishinde0723** [reply]
@AmiableAnil i did not find anything in network tab but getting this issue in console 
![image](https://github.com/Sunbird-Lern/Community/assets/72926048/0102d0a5-a23d-4d3f-82f3-5fe8900fd3ac)
